doc/API-Friendica.md

```diff
@@ -664,6 +660,23 @@ HTTP 403 Forbidden: when no authentication was provided
 HTTP 400 Bad Request: if given profile_id is not in the database or is not assigned to the authenticated user
 
 General description of profile data in API returns:
+- hide_friends: true if friends are hidden
+- profile_photo
+- profile_thumb
+- publish: true if published on the server's local directory
+- net_publish: true if published to global_dir
+- fullname
+- date_of_birth
+- description
+- xmpp
+- homepage
+- address
+- locality
+- region
+- postal_code
+- country
+- pub_keywords
+- custom_fields: list of public custom fields
```

Comment:
Shall we add the depracted fields too? At least it looks like based on this list:
https://github.com/friendica/friendica/pull/8163/files#diff-d414bc9a92f9e2057b85c44ffb50706bR87-R108

Reply:
*groans*

Reply:
nitpicking phil is at his best, I know .. If you say it isn't necessary it's OK for me too ;-) .. I just try to avoid accusations of tobias that we don't document well enough :D